src/main/java/teammates/storage/entity/FeedbackQuestion.java

```diff
@@ -145,19 +144,19 @@ public void setCourseId(String courseId) {
     }
 
     public String getQuestionMetaData() {
-        return questionText == null ? null : questionText.getValue();
+        return questionText == null ? null : questionText;
     }
 
     public void setQuestionText(String questionText) {
-        this.questionText = questionText == null ? null : new Text(questionText);
+        this.questionText = questionText == null ? null : questionText;
```

Comment:
For all these getters and setter you can just return or set as is, because there is no object wrapper anymore